x/evm/genesis.go

```diff
@@ -94,3 +98,19 @@ func ExportGenesis(ctx sdk.Context, k *keeper.Keeper, ak types.AccountKeeper) *t
 		Params:   k.GetParams(ctx),
 	}
 }
+
+var (
+	emptyCodeHash  = crypto.Keccak256(nil)
+	patchCodeHash1 = common.HexToHash("0x1d93f60f105899172f7255c030301c3af4564edd4a48577dbdc448aec7ddb0ac").Bytes()
```

Comment:
Sorry, not following what these two code hashes represent. Can you please explain?

Reply:
`0x6dbb3be328225977ada143a45a62c99ace929f536b75a27c09b6a09187dc70b0` is the codehash of the code `0x6d4946c0e9f43f4dee607b0ef1fa1c3314585733ff`, it contains `selfdestruct`
I am trying to dumping the code for the other one, I believe it's similar case, will update it in the PR.